Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/manager_app/view/plan_create_task_list/PlanTaskListFragment.java

```diff
@@ -4,16 +4,39 @@
 import android.app.FragmentTransaction;
 import android.databinding.DataBindingUtil;
 import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+
+import javax.inject.Inject;
+
+import database.entities.PlanTemplate;
+import database.repository.PlanTemplateRepository;
+import pg.autyzm.friendly_plans.ActivityProperties;
 import pg.autyzm.friendly_plans.App;
 import pg.autyzm.friendly_plans.R;
 import pg.autyzm.friendly_plans.databinding.FragmentPlanTaskListBinding;
 import pg.autyzm.friendly_plans.manager_app.view.plan_create_add_tasks.AddTasksToPlanFragment;
+import pg.autyzm.friendly_plans.manager_app.view.task_list.TaskRecyclerViewAdapter;
 
 public class PlanTaskListFragment extends Fragment implements PlanTaskListEvents {
 
+    private TaskRecyclerViewAdapter taskListAdapter;
+    private Long planId;
+
+    @Inject
+    PlanTemplateRepository planTemplateRepository;
+
+    TaskRecyclerViewAdapter.TaskItemClickListener taskItemClickListener =
+            new TaskRecyclerViewAdapter.TaskItemClickListener() {
+                @Override
+                public void onTaskItemClick(int position) {
+                    /* What to do after click? Remove task? Edit? */
```

Comment:
Change the order only, (but that's lots of work for different issue) deletion should be performed by clicking on special button in case somebody chooses it by mistake. I think editing task from this view will be confusing and will make this view too complicated than it could be - editing task assigned to plan will update same task in all plans, what could not be so obvious for all users.

Reply:
Ok. There is #277 issue for this. Let's leave it as it is - I'll remove this comment with the next PR.

Reply:
👍 Sure, I need more time to test it.

Reply:
I guess so :) This is not a small commit. :D